index.html

```diff
@@ -0,0 +1,150 @@
+            </div>
+            <h2 class="header-day-desktop">2023.7.7(SUN) ~ 13(SAT)</h2>
+            <h3 class="header-place-desktop">@ Stade de France, Concert and more</h3>
+        </div>
+        <div class="container-p">
+            <p class="header-p">If you are searching for a place to pass your vacations, and have curiosity whats happening in the party. This is your place, our main objective and principal because this project started is keep the young spirit, and spread the happines of the tecno music around the world, for ONLY ONE WEEK, what are you waiting for to come to the party!?</p>
+        </div>
+        <h2 class="header-day">2023.7.7(SUN) ~ 13(SAT)</h2>
+        <h3 class="header-place">@ Stade de France, Concert and more</h3>
+    </header>
+    <main class="main">
+        <section class="program-section">
+            <h2 class="program" id="program">Main Program</h2>
+            <div class="tiny-line"></div>
+            <div class="grid-container">
+                <div class="card"><img src="sources/mobile-version/images/mask-icon.png" alt="" class="card-img">
+                    <h2>thematic events</h2>
+                    <h3>Feel free to play as a character in the night of the cartoons, and roleplay in the other special events!</h3>
+                </div>
+                <div class="card"><img src="sources/mobile-version/images/tickets-icon.png" alt="" class="card-img">
+                    <h2>Free Bar & Entry</h2>
+                    <h3>You can ask for a free drink every day. As well get a free entry to one show in the last day of the event</h3>
+                </div>
+                <div class="card"><img src="sources/mobile-version/images/superstar-icon.png" alt="" class="card-img">
+                    <h2>Celebrities</h2>
+                    <h3>Famous musicians have going to participate in events all the days. Dont lose this oportunity!</h3>
+                </div>
+                <div class="card"><img src="sources/mobile-version/images/theme-park-icon.png" alt="" class="card-img">
+                    <h2>Theme Park</h2>
+                    <h3>You can visit diverse atractions all the day!Dont fogot go to the mountain russian!</h3>
+                </div>
+                <div class="card"><img src="sources/mobile-version/images/sun-icon.png" alt="" class="card-img">
+                    <h2>Non stop!</h2>
+                    <h3>This party dont gonna stop never! so you can be parting 24/7</h3>
+                </div>
+            </div>
+            <div class="button-container">
+            <button class="button-cards" id="join">Join to the party!</button>
+            </div>
+            <div class="grid-container-desktop">
+                <div class="card"><img src="sources/mobile-version/images/mask-icon.png" alt="" class="card-img">
```

Comment:
- [optional] please kindly centralize your cards in the main program section.
![Screenshot (1172)_LI](https://github.com/SaveryIV/First-Capstone-Project/assets/65298048/0a598104-3d9a-44ea-8974-379b3f8d4cd5)